src/tmpfiles/tmpfiles.c

```diff
@@ -2020,7 +2020,7 @@ static int create_device(Item *i, mode_t file_type) {
          * path can't be changed behind our back. */
         dfd = path_open_parent_safe(i->path, i->allow_failure);
         if (dfd < 0)
-                return dfd;
+                return log_error_errno(dfd, "Failed to open parent directory, not creating '%s': %m", i->path);
 
```

Comment:
i don't follow here. path_open_parent_safe() logs about all errors already except for chase() returning ENOLINK, for which the unsafe transition message is written. 

It appears to me, that you are only interested in the ENOLINK case here, judging by the commit message, so it appears to me you should change path_open_parent_safe() instead of create_device().

Moreover, the same applies to any row type, not just device creation anyway.

I think the unsafe thing makes a ton of sense in your case. Apparently the root dir of your fs tree is owned by non-root. That#s certainly an issue and specifically a security issue...

Reply:
I dropped the commit for now. I wrote this patch half a year ago and I don't quite remember what the conditions where this happened were.